docs/rest_apis.rst

```diff
@@ -201,6 +205,66 @@ Cloud verifier (CV)
     Stop cv polling on `agent_id`, but don’t delete (for an already started `agent_id`).
     This will make the agent verification fail.
 
+
+.. http:post::  /v2.1/allowlists/{ima_policy_name:string}
```

Comment:
While agreed in principle, as discussed in the team meeting today, we should hold off until a separate PR, since this is big/disruptive enough as is and the verifier APIs are not user-facing.